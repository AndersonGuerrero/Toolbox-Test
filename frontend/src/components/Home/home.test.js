import React from 'react'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import { Home } from './index'
import renderer from 'react-test-renderer'

configure({ adapter: new Adapter() })

test('Home render', () => {
	const component = renderer.create(
		<Home textdisplay={'Testing'} isLoading={false} />
	)
	let tree = component.toJSON()
	expect(tree).toMatchSnapshot()
})

test('Home Check Render', () => {
	const header = shallow(<Home textdisplay={'Testing'} isLoading={false} />)
	expect(header.find('h1').text()).toEqual('Testing')
})