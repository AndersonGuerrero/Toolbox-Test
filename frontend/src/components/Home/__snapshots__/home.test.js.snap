// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home render 1`] = `
<header
  className="masthead bg-primary text-white text-center"
>
  <div
    className="container d-flex align-items-center flex-column"
  >
    <img
      alt=""
      className="masthead-avatar mb-5"
      src="img/avataaars.svg"
    />
    <h1
      className="masthead-heading text-uppercase mb-0"
    >
      Testing
    </h1>
  </div>
</header>
`;
